src/PKSim.Core/Services/ProjectSnapshotToSimulationTransferMapper.cs

```diff
@@ -11,15 +11,15 @@
 
 namespace PKSim.Core.Services;
 
-public interface IProjectSnapshotToSimulationTransferMapper : IMapper<string, SimulationTransfer>;
+public interface IProjectSnapshotToSimulationTransferMapper : IMapper<string, (SimulationTransfer transfer, PKSimProject project)>;
 
 public class ProjectSnapshotToSimulationTransferMapper(
    IJsonSerializer jsonSerializer,
    ISnapshotMapper snapshotMapper,
    ISimulationConfigurationTask simulationConfigurationTask,
    ISimulationToModelCoreSimulationMapper simulationMapper) : IProjectSnapshotToSimulationTransferMapper
 {
-   public SimulationTransfer MapFrom(string snapshotString)
+   public (SimulationTransfer transfer, PKSimProject project) MapFrom(string snapshotString)
```

Comment:
unexpected return type here for a mapper

Reply:
We can make a class to hold them, but it would be single-use, I think.